doc/Programming-guide/Key-platform-features/Data-collection/File-system-log-appender/index.md

```diff
@@ -0,0 +1,235 @@
+
+```bash
+curl -v -S -u devuser:devuser123 -X POST -H 'Content-Type: application/json' -d @fileSystemLogAppender.json "http://localhost:8080/kaaAdmin/rest/api/logAppender" | python -mjson.tool
+```
+
+where file ```fileSystemLogAppender.json``` contains following data:
+
+```json
+{
+    "pluginClassName":"org.kaaproject.kaa.server.appenders.file.appender.FileSystemLogAppender",
+    "pluginTypeName":"File",
+    "applicationId":"5",
+    "applicationToken":"82635305199158071549",
+    "name":"Sample File system log appender",
+    "description":"Sample File system log appender",
+    "headerStructure":[
+        "KEYHASH",
+        "VERSION",
+        "TIMESTAMP",
+        "TOKEN",
+        "LSVERSION"
+    ],
+    "maxLogSchemaVersion":2147483647,
+    "minLogSchemaVersion":1,
+    "tenantId":"1",
+    "jsonConfiguration":"{\"publicKey\":\"public Key\",\"logsRootPath\":\"/kaa_log_uploads\",\"rollingFileNamePatern\":\"logFile.%d{yyyy-MM-dd}.log\",\"rollingMaxHistory\":30,\"triggerMaxFileSize\":\"1GB\",\"encoderPattern\":\"%-4relative [%thread] %-5level %logger{35} - %msg%n\"}"
+}
+```
+
+Example result:
+
+```json
+{
+    "applicationId":"5",
+    "applicationToken":"82635305199158071549",
+    "confirmDelivery":true,
+    "createdTime":1466154396923,
+    "createdUsername":"devuser",
+    "description":"Sample File system log appender",
+    "headerStructure":[
+        "KEYHASH",
+        "VERSION",
+        "TIMESTAMP",
+        "TOKEN",
+        "LSVERSION"
+    ],
+    "id":"131072",
+    "jsonConfiguration":"{\"publicKey\":\"public Key\",\"logsRootPath\":\"/kaa_log_uploads\",\"rollingFileNamePatern\":\"logFile.%d{yyyy-MM-dd}.log\",\"rollingMaxHistory\":30,\"triggerMaxFileSize\":\"1GB\",\"encoderPattern\":\"%-4relative [%thread] %-5level %logger{35} - %msg%n\"}",
+    "maxLogSchemaVersion":2147483647,
+    "minLogSchemaVersion":1,
+    "name":"Sample File system log appender",
+    "pluginClassName":"org.kaaproject.kaa.server.appenders.file.appender.FileSystemLogAppender",
+    "pluginTypeName":"File",
+    "tenantId":"1"
+}
+```
+
+# Playing with File system log appender
+
+Go to Data collection demos in Sandbox. And download binary.
+
+![Data collection demo in Sandbox](attach/data-collection-demo-in-sandbox.png)
+
+Next, in the Admin UI follow to **Data collection demo** application.
+
+Go to application's **Log appenders** configuration and add a new one.
+
+![Add log appender](attach/data-collection-demo-application.png)
+
+Enter name of the new appender (we use “Sample FileSystem log appender”)
+
+Add Log metadata fields.
+
+Select _File_ appender type.
+
+See [Creating file system log appender in Admin UI](#creating-file-system-log-appender-in-admin-ui) section for details.
+
+Verify that newly created appender has appeared in list.
+
+![Verify newly created log appender](attach/verify-created-appender.png)
+
+Now run Data collection demo application:
+
+```bash
+java -jar DataCollectionDemo.jar
```

Comment:
Could the command and output be split into separate code blocks to allow simple Copy & Paste?